store/sql_channel_store.go

```diff
@@ -992,7 +992,7 @@ func (s SqlChannelStore) GetForExport(teamId string) StoreChannel {
 		_, err := s.GetReplica().Select(&data, "SELECT * FROM Channels WHERE TeamId = :TeamId AND DeleteAt = 0 AND Type = 'O'", map[string]interface{}{"TeamId": teamId})
 
 		if err != nil {
-			result.Err = model.NewLocAppError("SqlChannelStore.GetAllChannels", "store.sql_channel.get_for_export.app_error", nil, "teamId="+teamId+", err="+err.Error())
+			result.Err = model.NewLocAppError("SqlChannelStore.GetForExport", "store.sql_channel.get_for_export.app_error", nil, "teamId="+teamId+", err="+err.Error())
```

Comment:
This PR: https://github.com/mattermost/platform/pull/3601 deletes this function. Revert this change to avoid conflict.